subsys/bluetooth/controller/ll_sw/openisa/hal/RV32M1/radio/radio.c

```diff
@@ -78,6 +84,14 @@ static u8_t MALIGN(4) _pkt_scratch[
 static s8_t rssi;
 
 
+#define RPA_TABLE_SIZE	8 /* Limit RPA Table to 8 entries */
```

Comment:
Use CONFIG_BT_CTLR_RL_SIZE? The range is 1 to 8

Reply:
Agreed, leverage the built in CONFIG values that provide the same information.

Reply:
I understand your point of view, I thought about using it in the first place. However, this define (RPA_TABLE_SIZE) refers to a limitation that we have for our underling HW block that helps us perform RPA, while define CONFIG_BT_CTLR_RL_SIZE refers to the software limitation.  Do you agree to have them split in this way or do you prefer to use CONFIG_BT_CTLR_RL_SIZE for both of them?

Reply:
Limiting based on HW support is fine. But that should happen in Kconfig in my opinion. There is a limit to the nRF AAR (HW RPA resolver) as well.